pkg/check/gc/reserve.go

```diff
@@ -70,32 +69,38 @@ func CheckReserve(c *bee.Cluster, o Options) error {
 	}
 	fmt.Println("reservestate:", state)
 
-	lowValueChunks := chunkBatch(rnd, overlay, int64(o.CacheSize), origState.Radius)
+	pinnedChunk := bee.GenerateRandomChunkAt(rnd, overlay, 0)
+	_, err = client.UploadChunk(ctx, pinnedChunk.Data(), api.UploadOptions{Pin: true, BatchID: batchID})
+	if err != nil {
+		return fmt.Errorf("unable to upload chunk: %w", err)
+	}
+
```

Comment:
@mrekucci can we add a `Printf` here with the pinned chunk address? similar to what we have in `L85` - `uploaded pinned chunk %s`...

Reply:
Done